system-variables.md

```diff
@@ -3663,6 +3663,17 @@ mysql> desc select count(distinct a) from test.t;
 - Default value: `1.0`
 - Indicates the net cost of transferring 1 byte of data through the network. This variable is internally used in the [Cost Model](/cost-model.md), and it is **NOT** recommended to modify its value.
 
+### tidb_opt_objective <span class="version-mark">New in v7.4.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Type: Enumeration
+- Default value: `moderate`
+- Possible values: `moderate`, `determinate`
+- This variable controls the objective of the optimizer. `moderate` maintains the default behavior from previous versions, which means the optimizer tries to make use of more information to generate better execution plans. While `determinate` mode tends to be more conservative and makes the execution plan more stable.
+- The real-time statistics are the total number of rows and the number of modified rows that are automatically updated according to DML statements. When this variable is set to `moderate`, the default value, TiDB will generate the execution plan based on the real-time statistics. When this variable is set to `determinate`, TiDB won't use the real-time statistics when generating the execution plan, which will make it more stable.
+- For long-term and stable OLTP workload, or the user is affirmative on the existing execution plans, the `determinate` mode is recommended to reduce the possibility of unexpected execution plan changes. The [`LOCK STATS`](/sql-statements/sql-statement-lock-stats.md) can also be used to prevent the statistics from being modified to further stabilize the execution plan.
```

Comment:
```suggestion
- For long-term stable OLTP workload, or if the user is affirmative on the existing execution plans, it is recommended to use the `determinate` mode to reduce the possibility of unexpected execution plan changes. Additionally, you can use the [`LOCK STATS`](/sql-statements/sql-statement-lock-stats.md) to prevent the statistics from being modified and further stabilize the execution plan.
```